assets/rich_text_components_specs.js

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable */
```

Comment:
Btw, I just realized, if this is going in assets, should we pull protractor.js out of extensions/rich_text_components as well and move it to somewhere in core/tests? Then delete extensions/rich_text_components altogether?

On the other hand, if extensions/rich_text_components is staying, then shouldn't this file go in it (like rule_templates.json goes in extensions/interactions)?

Reply:
We need _extensions/rich_text_components_ because there are other files we use. I would have to create new rule for this file in _app.yaml_ in order to serve it from _extensions/rich_text_components_

Reply:
Hm. How does extensions/interactions/rule_templates.json do it?

Reply:
This file is not directly loaded from the frontend.

Reply:
I see, OK. In that case, let's keep it in assets/ and put a README.md file in extensions/rich_text_components that points to it.

Reply:
Done.